packages/loader/driver-utils/api-report/driver-utils.api.md

```diff
@@ -220,7 +220,7 @@ export interface IProgress {
 }
 
 // @internal
-export function isCombinedAppAndProtocolSummary(summary: ISummaryTree | undefined, ...optionalRootTrees: string[]): summary is CombinedAppAndProtocolSummary;
```

Comment:
Maybe we need to keep the optionalRootTrees making them optional for back compat?